### PR TITLE
Fix generator config

### PR DESCRIPTION
**Pull request by MaxGfeller**
Small fixes to the generator config.

### CODE REVIEW

File: gen_config/model.conf [hunk 1]
```diff
@@ -1 +1 @@
-java -cp target/killbill-java-parser-0.0.1-SNAPSHOT.jar com.ning.killbill.KillbillParser -i file:///<KILLBILL_REPO>/jaxrs/src/main/java/com/ -o /tmp/output -t PHP_CLIENT_API -p com.ning.billing.jaxrs.json
+java -cp target/killbill-java-parser-0.0.1-SNAPSHOT.jar com.ning.killbill.KillbillParser -i file://<KILLBILL_REPO>/jaxrs/src/main/java/org -o /tmp/output -t PHP_CLIENT_API -p org.killbill.billing.jaxrs.json
```
Comment (pierre) [inline gen_config/model.conf]: Indeed, we changed the namespace quite a few years back 😸 

However, looking at the script that I use locally (just verified it still runs correctly), I do have `file:///$BASE` (3 `/`) where `BASE=/foo/bar/baz/killbill/jaxrs/src/main/java/` (no `org`).

Your version may work but that way we would be consistent with others (e.g. [Ruby](https://github.com/killbill/killbill-client-ruby/blob/master/gen_config/model.conf) one).

Comment (MaxGfeller) [inline gen_config/model.conf]: But this is pointing to the main Killbill repo, no?

The path there is with `org/` (or am i missing something?): https://github.com/killbill/killbill/tree/master/jaxrs/src/main/java/org

Comment (pierre) [inline gen_config/model.conf]: Yup, but it looks like you can put either `killbill/jaxrs/src/main/java/` or `killbill/jaxrs/src/main/java/org`, the code is smart enough to find the files regardless.

Same for `file://` vs `file:///`, code supports both.

Anyways, not really important, it was just more a consistency thing.

**Comment by pierre** [comment]
Thanks!

File: gen_config/model.conf (hunk 1 above)
Comment (MaxGfeller) [inline gen_config/model.conf]: Oh, i see. Thanks for merging! :)

**Pull request merged**